test/controllers/IndexControllerSpec.scala

```diff
@@ -39,5 +39,23 @@ class IndexControllerSpec extends SpecBase {
 
       application.stop()
     }
+
+    "redirect to AgentOverview with Agent affinityGroup for a GET" in {
+
+      val application = applicationBuilder(userAnswers = None).build()
+
+      val request = FakeRequest(GET, routes.IndexController.onPageLoad().url).withSession("affinityGroup" -> "Agent")
```

Comment:
We cannot set `.withSession` as explained above.

Reply:
Done